### ISSUE TITLE
404 from the put command

### ISSUE THREAD

**Issue by kostikbel** [s-aska/dropbox-api-command#31]
I am trying to upload large (> 2G) files to dropbox folder and get the following errors

```
tom% TMPDIR=/var/tmp dropbox-api put /usr/home/backups/c-tom-backup.16-10-11_03:30:00.tbz dropbox:/tom/c-tom-backup.16-10-11_03:30:00.tbz
2016-10-11T17:54:25 [WebService::Dropbox] [ERROR] https://content.dropboxapi.com/2/files/upload {"path":"/tom/c-tom-backup.16-10-11_03:30:00.tbz","mode":"overwrite"} -> [404] <html>
<head><title>404 Not Found</title></head>
<body bgcolor="white">
<center><h1>404 Not Found</h1></center>
<hr><center>nginx</center>
</body>
</html>
<html>
<head><title>404 Not Found</title></head>
<body bgcolor="white">
<center><h1>404 Not Found</h1></center>
<hr><center>nginx</center>
</body>
</html>
```

Sometimes, the upload succeeds, but this is very rare event, say 1 in 20 tries.  For short files, the put command seems to work reliably.

dropbox-api 2.09, perl 5.24 rc3, freebsd 9/stable amd64.

**Comment by s-aska** [comment]
Fixed and released v2.12.
I think that this treatment was cured.

**Comment by kostikbel** [comment]
It seems that the problem is solved indeed, thank you.
